src/redturtle/prenotazioni/profiles/default/types/BookingType.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        meta_type="Dexterity FTI"
+        name="BookingType"
```

Comment:
I would change the name in PrenotazioneType just for consistency with the other ones

Reply:
I think we need to move in the english names direction every time we are going to modify this project

Reply:
you can't do it with object names and classes, because you need to change the class of created objects, so let's stay with this fancy naming, but use the same convention